packages/components/slider/tests/range-slider.test.tsx

```diff
@@ -183,48 +177,48 @@ describe("<RangeSlider />", () => {
 
     let sliderThumb = sliderThumbs[1]
 
-    await act(() => fireEvent.focus(sliderThumb))
-    await act(() => fireEvent.keyDown(sliderThumb, { key: "ArrowRight" }))
+    sliderThumb.focus()
+    await userEvent.keyboard("{ ArrowRight }")
```

Comment:
@neginasu 
You can obtain a `user` object from the return value of `render()`. Please use this user instead of `userEvent`.

Correct:

```tsx
const { container, user } = render(...)

// ...

await user.keyboard("{ArrowRight}") 
```

Reply:
@koralle 
Thank you.
I will do that when I call userEvent.